setup/environment.yml

```diff
@@ -1,428 +1,479 @@
+  - tinycss2=1.2.1=pyhd8ed1ab_0
+  - tk=8.6.12=h27826a3_0
+  - tktable=2.10=hb7b940f_3
+  - toml=0.10.2=pyhd8ed1ab_0
+  - tomli=2.0.1=pyhd8ed1ab_0
+  - toolz=0.12.0=pyhd8ed1ab_0
+  - tornado=6.3.2=py38h01eb140_0
+  - tqdm=4.65.0=pyhd8ed1ab_1
+  - traitlets=5.9.0=pyhd8ed1ab_0
+  - typing-extensions=4.6.3=hd8ed1ab_0
+  - typing_extensions=4.6.3=pyha770c72_0
+  - typing_utils=0.1.0=pyhd8ed1ab_0
+  - tzdata=2023c=h71feb2d_0
+  - tzlocal=5.0.1=py38h578d9bd_0
+  - ucx=1.14.1=hf587318_2
+  - ukkonen=1.0.1=py38h43d8883_3
+  - umap-learn=0.5.3=py38h578d9bd_1
+  - unicodedata2=15.0.0=py38h0a891b7_0
+  - urllib3=1.26.15=pyhd8ed1ab_0
+  - vcfanno=0.3.3=h9ee0642_0
+  - virtualenv=20.23.0=pyhd8ed1ab_0
+  - wcwidth=0.2.6=pyhd8ed1ab_0
+  - webencodings=0.5.1=py_1
+  - websocket-client=1.5.3=pyhd8ed1ab_0
+  - wheel=0.40.0=pyhd8ed1ab_0
+  - widgetsnbextension=4.0.7=pyhd8ed1ab_0
+  - xcb-util=0.4.0=hd590300_1
+  - xcb-util-image=0.4.0=h8ee46fc_1
+  - xcb-util-keysyms=0.4.0=h8ee46fc_1
+  - xcb-util-renderutil=0.3.9=hd590300_1
+  - xcb-util-wm=0.4.1=h8ee46fc_1
+  - xkeyboard-config=2.39=hd590300_0
+  - xlrd=2.0.1=pyhd8ed1ab_3
+  - xorg-fixesproto=5.0=h7f98852_1002
+  - xorg-inputproto=2.3.2=h7f98852_1002
+  - xorg-kbproto=1.0.7=h7f98852_1002
+  - xorg-libice=1.0.10=h7f98852_0
+  - xorg-libsm=1.2.3=hd9c2040_1000
+  - xorg-libx11=1.8.6=h8ee46fc_0
+  - xorg-libxau=1.0.11=hd590300_0
+  - xorg-libxdmcp=1.1.3=h7f98852_0
+  - xorg-libxext=1.3.4=h0b41bf4_2
+  - xorg-libxfixes=5.0.3=h7f98852_1004
+  - xorg-libxi=1.7.10=h7f98852_0
+  - xorg-libxrender=0.9.10=h7f98852_1003
+  - xorg-libxt=1.3.0=hd590300_0
+  - xorg-libxtst=1.2.3=h7f98852_1002
+  - xorg-recordproto=1.14.2=h7f98852_1002
+  - xorg-renderproto=0.11.1=h7f98852_1002
+  - xorg-xextproto=7.3.0=h0b41bf4_1003
+  - xorg-xf86vidmodeproto=2.3.1=h7f98852_1002
+  - xorg-xproto=7.0.31=h7f98852_1007
+  - xyzservices=2023.5.0=pyhd8ed1ab_1
+  - xz=5.2.6=h166bdaf_0
+  - yaml=0.2.5=h7f98852_2
+  - yarl=1.9.2=py38h01eb140_0
+  - zeromq=4.3.4=h9c3ff4c_1
+  - zict=3.0.0=pyhd8ed1ab_0
+  - zipp=3.15.0=pyhd8ed1ab_0
+  - zlib=1.2.13=hd590300_5
+  - zlib-ng=2.0.7=h0b41bf4_0
+  - zstd=1.5.2=h3eb15da_6
   - pip:
-    - arrow==1.1.0
-    - binaryornot==0.4.4
-    - jinja2-time==0.2.0
-    - libcst==0.3.23
-    - memory-profiler==0.55.0
-    - pathspec==0.9.0
-    - poyo==0.5.0
-    - pycln==1.2.5
-    - python-slugify==5.0.2
-    - setuptools==50.3.2
-    - simppl==1.0.7
-    - sklearn-pandas==1.8.0
-    - text-unidecode==1.3
-    - typer==0.4.0
-    - typing-inspect==0.7.1
-prefix: /home/ubuntu/miniconda3/envs/genomics.py3
+      - about-time==4.2.1
+      - alive-progress==3.1.4
+      - grapheme==0.6.0
+      - mpld3==0.5.9
+      - pdf2image==1.16.3
+      - pymupdf==1.22.5
+      - pypdf2==3.0.1
+      - reportlab==4.0.4
+      - sigprofilerassignment==0.0.30
+      - sigprofilermatrixgenerator==1.2.17
+      - sigprofilerplotting==1.3.14
+      - simppl==1.0.7
+prefix: /home/ilyas/anaconda3/envs/genomics.py3
```

Comment:
Might be best to delete this last line

https://stackoverflow.com/questions/41274007/anaconda-export-environment-file

Reply:
Fixed